test/Microsoft.SqlTools.ServiceLayer.UnitTests/ObjectExplorer/ObjectExplorerServiceTests.cs

```diff
@@ -56,7 +56,7 @@ public void InitObjectExplorerServiceTests()
             ConnectedBindingContext connectedBindingContext = new ConnectedBindingContext();
             connectedBindingContext.ServerConnection = new ServerConnection(new SqlConnection(fakeConnectionString));
             connectedBindingQueue = new ConnectedBindingQueue(false);
-            connectedBindingQueue.BindingContextMap.Add($"{details.ServerName}_{details.DatabaseName}_{details.UserName}_NULL", connectedBindingContext);
+            connectedBindingQueue.BindingContextMap.Add($"{details.ServerName}_{details.DatabaseName}_{details.UserName}_NULL_persistSecurityInfo:true", connectedBindingContext);

```

Comment:
Any special reason for this change?

Reply:
The PersistSecurityInfo is an option that is added as a result of taking into every property besides password and connection name (as it's not supported by SQL connections) to the URI.

It is considered a non default option when it is true on the ADS side, and since we want to treat non special options consistently, it will have to be added to the string here.

Reply:
The PersistSecurityInfo is an option that is added as a result of taking into every property besides password and connection name (as it's not supported by SQL connections) to the URI.

It is considered a non default option when it is true on the ADS side, and since we want to treat non special options consistently, it will have to be added to the string here.